3_javascript/javascript3/week2/calculator.js

```diff
@@ -0,0 +1,52 @@
+
+async function fetchExchangeRates() {
+    try {
+        const response = await fetch('https://open.er-api.com/v6/latest/USD');
+        const data = await response.json();
+        const exchangeRates = data.rates;
+        const currencies = Object.keys(exchangeRates);
+
+        const fromCurrencySelect = document.getElementById('fromCurrency');
+        const toCurrencySelect = document.getElementById('toCurrency');
+
+
+        currencies.forEach((currency) => {
```

Comment:
Nice idea to use ```forEach``` to loop through the array. However, this could have been part of a function to avoid repetitive code.